docs-source/content/samples/simple/tanzu-kubernetes-service/_index.md

```diff
@@ -639,64 +639,42 @@ $ helm install ingress-nginx ingress-nginx/ingress-nginx
 ```
 
 
-Create ingress routing for accessing the application deployed in the cluster.
+Create ingress for accessing the application deployed in the cluster and to access Administration console.
```

Comment:
to access Administration console. -> to access the Administration Console.

Reply:
fixed